validator/src/main/java/com/google/daq/mqtt/sequencer/sequences/ConfigSequences.java

```diff
@@ -153,4 +153,24 @@ public void extra_config() {
     untilLogged(SYSTEM_CONFIG_APPLY, SYSTEM_CONFIG_APPLY_LEVEL);
   }
 
+  @Test(timeout = ONE_MINUTES_MS)
+  @Feature(stage = BETA, bucket = SYSTEM)
+  @Summary("Check that the device publishes minimum required log entries when receiving config")
+  public void config_logging() {
+    deviceConfig.system.min_loglevel = Level.DEBUG.value();
+    updateConfig("send device config set loglevel to debug");
+
+    // Wait for device to receive the config
+    // We can untilLogged the system.config.apply event, but following the natural ordering
+    // of receive --> parse --> apply may be more helpful
```

Comment:
may?  I don't understand this explanation.  Seems unnecessary...

Reply:
Removed